### ISSUE TITLE
Italian pagetotal localization

### ISSUE THREAD

**Issue by fargiolas** [plk/biblatex#689]
Hi, since #534 and #546, if an entry has a pagetotal field and I'm using italian localization with polyglossia I get bold pagetotals strings in the bibliography and undefined string warning in the logs.

As far I know there's no difference in Italian with putting pages string before or after the number, so _pagetotal_ and _pagetotals_ should be translated with the same strings as _page_ and _pages_:

```
pagetotal             = {{pagina}{p\adddot}},
pagetotals            = {{pagine}{pp\adddot}},
```

I was wondering, how were these fields handled before that commit? if they were silently mapped to _page_ and _pages_ this could be a regression for some language. I'm not sure assuming a possibly wrong translation would be better, maybe do it but still give a warning? or keep the current behavior but give a better warning for strings with missing translation, something like _Please consider contributing a translation at..._

See also [this question](https://tex.stackexchange.com/questions/411681/biblatex-pagetotals-localization) on stackexchange.

**Comment by moewew** [comment]
Thank you for reporting this. Can you check if the other `...total` strings are also the same?

https://github.com/plk/biblatex/blob/9f6b217d71f13d0b0e6dc09fb5fadc9e5d3ce7de/tex/latex/biblatex/lbx/italian.lbx#L296-L319

If you have a lot of time on your hands we would appreciate a full look at the state of the entire `italian.lbx`. A few other strings are also missing.

Before the commit there were no `...total` strings and their non-`total` counterparts were used instead. Yes this is a regression for some languages, but I rather have a bold bibstring and a warning that clearly shows that something is missing than a wrong translation that can go unnoticed. Falling back to `pages` would have been possible, but fiddly, and I'm not sure all users read the warnings (experience teaches me many don't).

I like the idea of making the warning a bit more explicit though.

**Comment by fargiolas** [comment]
> and I'm not sure all users read the warnings (experience teaches me many don't)

You're right, a lot of cruft can build up in the logs with big documents, it went unnoticed for me too until I isolated the issue in a minimal example.

I'm a bit busy at the moment, will try to look a the other strings later today.

**Comment by fargiolas** [comment]
@moewew I translated a couple of strings in my fork, not sure it's worth a pull request though. Please see 71b2637390935d942a5477ea562164bc396af114 and 06cff5516959fd87ab42cc1b64fb751f6d8eb201

**Comment by moewew** [comment]
It definitely is worth one. Just submit a pull request to `dev` and I will merge. If you want to appear in the list of language contributors in the documentation I'll add you if you tell me how you want to appear.

**Comment by moewew** [comment]
Merged in #691. Thank you very much.

**Comment by fargiolas** [comment]
thanks to you, don't worry about crediting me, it's just a trivial contribution
